interfaces/seccomp/backend.go

```diff
@@ -58,35 +57,38 @@ var (
 	releaseInfoId            = release.ReleaseInfo.ID
 	releaseInfoVersionId     = release.ReleaseInfo.VersionID
 	requiresSocketcall       = requiresSocketcallImpl
-	seccompCompilerLookup    = seccompToBpfPath
+	snapSeccompVersionInfo   = snapSeccompVersionInfoImpl
```

Comment:
can we split this two in their own var grouping,  I didn't notice before but is a bit odd to mix external ones here and this two which are defined her